src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -57,9 +57,9 @@
   public static final String DEFAULT_POLINE_LIMIT = "1";
   private static final String MAX_POLINE_LIMIT = "500";
   public static final String OKAPI_URL = "X-Okapi-Url";
-  public static final String PO_LINES_LIMIT_PROPERTY = "poLines-limit";
+  private static final String PO_LINES_LIMIT_PROPERTY = "poLines-limit";
```

Comment:
why does this need to be private?

Reply:
The only reason is that this constant is not used anywhere except in this class.

Reply:
I guess it's a personal preference, but I find it silly to limit scope of static constants as there's no benefit IMO.  Additionally, if you ever want to refer to them later on (e.g. from a unit test or other class) it would require changing the scope again.